### ISSUE TITLE
SATOSHIPORTAL-WORKFLOW.md is empty

### ISSUE THREAD

**Issue by justinmoon** [SatoshiPortal/cyphernode#20]
[This would be a very interesting read](https://github.com/SatoshiPortal/cyphernode/blob/master/doc/SATOSHIPORTAL-WORKFLOW.md)

Making an issue to suggest you either fill this out or remove it.

2 cents ...

**Comment by Kexkey** [comment]
Hey Justin!  You’re right.  But they’re coming.  Workflow file and also the manifest...

And... we had a blast in Riga, let’s do it again!  😊

**Comment by FrancisPouliot** [comment]
Youre right, I forgot to commit the files. I was still editing them 10 minutes before we opened the repo 😅 - coming iminently
